nixos/modules/services/misc/calibre-server.nix

```diff
@@ -94,8 +110,8 @@ in
         };
 
         userDb = lib.mkOption {
-          default = null;
-          type = lib.types.nullOr lib.types.path;
+          default = "";
```

Comment:
Unrelated. Please factor out to a separate commit.

Reply:
Done